### ISSUE TITLE
create an extension for PostgreSQL

### ISSUE THREAD

**Issue by bartvde** [tudelft3d/prepair#3]
Thanks for your presentation at #osgeonl today.

I talked to my colleague Paul Ramsey about how to integrate this into PostGIS, and he came up with an alternative. See below:

Their code could be an extension that consumes WKB and outputs WKB
Requires no dependency on postgis at compile time
And just has a couple SQL functions that take in geometry, output wkb, hand to the internal functions, etc
Basically, that's how I did the pgpointcloud postgis integration
Pointcloud has no compile-time dependency on postgis
And all the interaction is via wkb in bytea structures
Makes it easy to write add-ons without the overhead of getting into the postgis build
